Changelog.md

```diff
@@ -344,10 +344,10 @@ Enhancements:
 * Improve controller and routing spec calls to `routes` by using `yield`
   instead of `call`. (Anton Davydov, #1308)
 * Add support for `ActiveJob` specs as standard `RSpec::Rails::RailsExampleGoup`s
-  via both `:type => :job` and inferring type from spec directory `spec/jobs`.
+  via both `type: :job` and inferring type from spec directory `spec/jobs`.
```

Comment:
Please rollback Changelog changes, it's unnecessary.

Reply:
This still needs to happen.

Reply:
Decided it was ok